qa/smoke-test-reindex-with-groovy/pom.xml

```diff
@@ -0,0 +1,272 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>elasticsearch-qa</artifactId>
+        <groupId>org.elasticsearch.qa</groupId>
+        <version>2.3.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <!--
+      This test unzips and starts elasticsearch.
+      It then uses rest tests to test reindex and groovy together!
+    -->
+
+    <artifactId>smoke-test-reindex-with-groovy</artifactId>
```

Comment:
Oh! Not just rest tests. This is like 10 lines in master with gradle. In 2.x in is a copy and paste fest.